source/jobs.html.erb

```diff
@@ -20,7 +20,15 @@ unboxed_meeting_image:
   image_description: Unboxers working together on code
 roles:
   title: Roles we are looking to fill
-  content:
+  content: 
+  - title: Senior product designer
+    name: Senior product designer
+    text: "We are on the lookout for a London-based Senior Product Designer, with solid experience of designing user flows and prototypes, who loves working with teams to turn designs into digital products that make a meaningful impact in the world."
+    link: /jobs/senior-product-designer
+  - title: Senior developer
+    name: Senior developer
+    text: "We are on the lookout for Ruby on Rails Developers to join us in London. As a core part of our team, our developers are responsible for building scalable, sustainable systems from project inception right through to production."
+    link: /jobs/developer
```

Comment:
This should link to the senior developer job. I think there's a file in there called developer_senior_developer or something like that. Probably best to edit that to make it just a senior developer job role.